{"name": "test_login_success", "status": "broken", "statusDetails": {"message": "AttributeError: 'LoginPage' object has no attribute 'is_logout_text_exist'", "trace": "self = <cases.test_login.TestLogin object at 0x0430C0D0>\nhandler = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"b174d39c8f052119233df31bffaf1c9e\")>, <pages.login_page.LoginPage object at 0x04048F70>)\n\n    @pytest.mark.last\n    def test_login_success(self,handler):\n        #sys._getframe().f_code.co_name : 获取test_login_success，且打印出\n        logger.info('执行{}测试用例'.format(sys._getframe().f_code.co_name))\n        logger.info('进入 test_login_success methon')\n        logger.info(\"登录测试—正常数据\")\n        handler[1].clear_text(LoginLocator.username_loc)  # 清空输入框内容\n        handler[1].clear_text(LoginLocator.password_loc)  # 清空输入框内容\n        # 输入正确的用户名，密码进行登录，其中handler【1】表示lg（即，driver.get(login_url) ）\n        handler[1].login(success_data['username'],success_data['password'])\n        logger.info('期望值：{}'.format(True))\n>       logger.info('实际值：{}'.format(handler[1].is_logout_text_exist()))  #is_logout_text_exist()\nE       AttributeError: 'LoginPage' object has no attribute 'is_logout_text_exist'\n\ncases\\test_login.py:24: AttributeError"}, "start": 1609587149546, "stop": 1609587165474, "uuid": "3617476b-a0e1-4b03-96d6-4bd2d173cad9", "historyId": "4de4eac7aa5437331edc4ae996ff58ee", "testCaseId": "a3db9ab94b753eb349ff93c7e366cb8a", "fullName": "cases.test_login.TestLogin#test_login_success", "labels": [{"name": "tag", "value": "last"}, {"name": "tag", "value": "run(order=-1)"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "8060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}